apps/src/sites/studio/pages/layouts/_thank_donors_interstitial.js

```diff
@@ -4,7 +4,7 @@ import firehoseClient from '@cdo/apps/lib/util/firehose';
 
 $(document).ready(function() {
   var already_shown = !!cookies.get('has_seen_thank_donors');
-  if (!already_shown && window.innerWidth > 720) {
+  if (!already_shown && window.innerWidth > 800 && window.innerHeight > 600) {
```

Comment:
The modal is ~700x500, so somewhat arbitrary decision to show on devices where there'd be a bit of padding. I want young kids to be able to dismiss it even if they can't read it, which will happen when they click/touch anywhere outside of the modal (or click/touch the button on the modal).